docs/en/04-supported-codebases/mmdet.md

```diff
@@ -1,30 +1,175 @@
+
+## Convert model
+
+You can use [tools/deploy.py](https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/tools/deploy.py) to convert mmdet models to the specified backend models. Its detailed usage can be learned from [here](https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/02-how-to-run/convert_model.md#usage).
+
+The following shows an example about converting `faster r-cnn` model to onnx model that can be inferred by ONNX Runtime.
+
+```shell
+cd mmdeploy
+# download faster r-cnn model from mmdet model zoo
+mim download mmdet --config faster-rcnn_r50_fpn_1x_coco --dest .
+# convert mmdet model to onnxruntime model with dynamic shape
+python tools/deploy.py \
+    configs/mmdet/detection/detection_onnxruntime_dynamic.py \
+    faster-rcnn_r50_fpn_1x_coco.py \
+    faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
+    tests/data/tiger.jpeg \
+    --work-dir mmdeploy_models/mmdet/ort \
+    --device cpu \
+    --show \
+    --dump-info
+```
+
+It is crucial to specify the correct deployment config during model conversion. We've already provided builtin deployment config [files](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmdet) of all supported backends for mmdetection, under which the config file path follows the pattern:
+
+```
+{task}/{task}_{backend}-{precision}_{static | dynamic}_{shape}.py
+```
+
+- **{task}:** task in mmdetection.
+
+  There are two of them. One is `detection` and the other is `instance-seg`, indicating instance segmentation.
+
+  mmdet models like `RetinaNet`, `Faster R-CNN` and `DETR` and so on belongs to `detection` task. While `Mask R-CNN` is one of `instance-seg` models. You can find more of them in chapter [Supported models](#supported-models).
+
+  **DO remember to use** `detection/detection_*.py` deployment config file when trying to converting detection models and use `instance-seg/instance-seg_*.py` to deploy instance segmentation models.
+
+- **{backend}:** inference backend, such as onnxruntime, tensorrt, pplnn, ncnn, openvino, coreml and etc.
+
+- **{precision}:** fp16, int8. When it's empty, it means fp32
+
+- **{static | dynamic}:** static shape or dynamic shape
+
+- **{shape}:** input shape or shape range of a model
+
+Therefore, in the above example, you can also convert `faster r-cnn` to other backend models by changing the deployment config file `detection_onnxruntime_dynamic.py` to [others](https://github.com/open-mmlab/mmdeploy/tree/dev-1.x/configs/mmdet/detection), e.g., converting to tensorrt-fp16 model by `detection_tensorrt-fp16_dynamic-320x320-1344x1344.py`.
+
+```{tip}
+When converting mmdet models to tensorrt models, --device should be set to "cuda"
+```
+
+## Model Specification
+
+Before moving on to model inference chapter, let us talk more about the converted model structure which is very important to do model inference.
```

Comment:
let us talk more about the converted model structure which is very important to do model inference. -> let's know more about the converted model structure which is important for model inference.